lib/ansible/modules/network/fortios/fortios_firewall_profile_protocol_options.py

```diff
@@ -0,0 +1,927 @@
+                "pop3": {"required": False, "type": "dict",
+                         "options": {
+                             "inspect-all": {"required": False, "type": "str",
+                                             "choices": ["enable", "disable"]},
+                             "options": {"required": False, "type": "str",
+                                         "choices": ["fragmail", "oversize"]},
+                             "oversize-limit": {"required": False, "type": "int"},
+                             "ports": {"required": False, "type": "int"},
+                             "scan-bzip2": {"required": False, "type": "str",
+                                            "choices": ["enable", "disable"]},
+                             "status": {"required": False, "type": "str",
+                                        "choices": ["enable", "disable"]},
+                             "uncompressed-nest-limit": {"required": False, "type": "int"},
+                             "uncompressed-oversize-limit": {"required": False, "type": "int"}
+                         }},
+                "replacemsg-group": {"required": False, "type": "str"},
+                "rpc-over-http": {"required": False, "type": "str",
+                                  "choices": ["enable", "disable"]},
+                "smtp": {"required": False, "type": "dict",
+                         "options": {
+                             "inspect-all": {"required": False, "type": "str",
+                                             "choices": ["enable", "disable"]},
+                             "options": {"required": False, "type": "str",
+                                         "choices": ["fragmail", "oversize", "splice"]},
+                             "oversize-limit": {"required": False, "type": "int"},
+                             "ports": {"required": False, "type": "int"},
+                             "scan-bzip2": {"required": False, "type": "str",
+                                            "choices": ["enable", "disable"]},
+                             "server-busy": {"required": False, "type": "str",
+                                             "choices": ["enable", "disable"]},
+                             "status": {"required": False, "type": "str",
+                                        "choices": ["enable", "disable"]},
+                             "uncompressed-nest-limit": {"required": False, "type": "int"},
+                             "uncompressed-oversize-limit": {"required": False, "type": "int"}
+                         }},
+                "switching-protocols-log": {"required": False, "type": "str",
+                                            "choices": ["disable", "enable"]}
+
+            }
+        }
+    }
+
+    module = AnsibleModule(argument_spec=fields,
+                           supports_check_mode=False)
+    try:
+        from fortiosapi import FortiOSAPI
+    except ImportError:
```

Comment:
Once all your current PRs have been merged you may wish to update raise a single PR to bulk update the Import check to the form https://github.com/ansible/community/issues/346#issuecomment-461037623

Reply:
Thanks @gundalow. Yes, we will raise a new PR to cover all of them for sure. Actually it is a very good point to improve usability for the end user.